chalicelib_fourfront/checks/wrangler_checks.py

```diff
@@ -512,12 +527,18 @@ def biorxiv_version_update(connection, **kwargs):
     current_biorxivs = ff_utils.search_metadata(query, key=connection.ff_keys)
 
     items_to_update = []
+    problem_ids = []
     biorxiv_api = 'https://api.biorxiv.org/details/biorxiv/'
     for publication in current_biorxivs:
-        if not publication['ID'].startswith('doi:'):
+        pubid = publication.get('ID')
+        if not pubid.startswith('doi:'):
+            continue
+        doi, misformatted = _remove_prefix_and_version_suffix_from_doi(publication['ID'])
```

Comment:
Pretty minor, but since you have `pubid`, can replace `publication['ID']` (i.e. `_remove_prefix_and_version_suffix_from_doi(pubid)`)